desktop/api/src/main/java/org/datacleaner/util/WidgetUtils.java

```diff
@@ -728,4 +692,7 @@ public static void setDefaultButtonStyle(final AbstractButton button) {
         }
     }
 
+    private static int pix(int p) {
```

Comment:
Okay, not 100% happy, though very minor. Missing final (and of course the one-char name from Travis) :)